brainrender/camera.py

```diff
@@ -0,0 +1,102 @@
+import vtk
+
+from .cameras import cameras
+
+
+def get_camera(camera):
```

Comment:
It looks like this module and the context of `cameras.py` could be refactored in a nice `Camera` class!

Reply:
I prefer to leave it as it is for now, but I will think about it. 

Almost always users will want to either pass a camera name or a dictionary of parameters to `scene.render`, everything else is just internal checks to make sure that camera arguments are correct.
If there was the need to expose more camera functionality to the user it would make sense to turn it into a `Camera` class, but as such there wouldn't be much benefit. 

This approach is still quite flexible (e.g. the new `Animation` class interpolates between two cameras) and (I hope) not too confusing for developers.

Reply:
even if you don't expose it it could be helpful for reading the code. But yeah, no priority, until you want people to use it